json_spec/output/dut_info.json

```diff
@@ -39,7 +39,7 @@
         }
     },
     "required": [
-        "id"
+        "dutInfoId"
```

Comment:
while i get the problem here, this should not be in this PR; id like to avoid clobbering unrelated code changes, esp stuff that touches the spec

Reply:
This is related. I turned on the CI and fixed a bug in the spec. This is not a change in the spec, this is an error. It's requiring a field that doesn't exist. This PR can't be submitted without this change. I'm happy to update the description for this PR to reflect the change better, but creating a second PR for a one line code change is a bunch of unecessary work that accomplishes nothing IMO.

Reply:
ok, valid argument; i'll approve